stylesheets/app.scss

```diff
@@ -39,6 +41,11 @@
     .chat-input {
       @include span-columns(10 of 10);
     }
-    
+
+    .new-message {
+      border-top: 2px solid blue;
```

Comment:
You can use $blue (or pick a different color in variables) to keep the color scheme consistent